### PR TITLE
EOS-17432: Unexplained "cpu|fault" and "psu_fault|resolved" events

### PR DESCRIPTION
**Pull request by soniyamoholkar**
# Backend

## Problem Statement
<pre>
Story Ref (if any): EOS-17432: Unexplained "cpu|fault" and "psu_fault|resolved" events
</pre>
## Unit testing on RPM done
<pre>
Yes
</pre>
## Problem Description
<pre>
Unexplained "cpu|fault" and "psu_fault|resolved" events
</pre>
## Solution
<pre>
Updated the description field mapping as per SSPL description.
</pre>
## Unit Test Cases
<pre>

</pre>
Signed-off-by:

### CODE REVIEW

**Comment by gowthamchinna** [comment]
test this please

**Comment by gowthamchinna** [comment]
test this please

File: csm/plugins/cortx/alert.py [hunk 1]
```diff
@@ -298,9 +298,20 @@ def _convert_to_csm_schema(self, message):
...
                 alert has come.
+                6. Enclosure alerts can be generated from any node. So one alert
+                is generated from one node and another alert for the same enclosure
+                componenet is generated from another node, the status should get
+                updated accordingly. That is why node_id is ignored in case of
+                enclosure while creating sensor_info.
                 """
-                csm_schema[const.ALERT_SENSOR_INFO] = \
-                    '_'.join(str(x) for x in csm_schema[const.ALERT_SENSOR_INFO].values())
+                if res_split[0] == const.ENCLOSURE:
```
Comment (pawankumarsrivastava) [inline csm/plugins/cortx/alert.py:307]: @kupranay Is this required for R2 since we will only have 1:1 mapping for node server and storage_encl?

Comment (kupranay) [reply csm/plugins/cortx/alert.py:307]: Good point.. this is not required for R2. @soniyamoholkar  please remove it

Comment (soniyamoholkar) [reply csm/plugins/cortx/alert.py]: @kupranay and @pawankumarsrivastava  Removed the check.

**Review by pawankumarsrivastava — APPROVED** [review]

**Review by kupranay — APPROVED** [review]

**Pull request merged**